avalon/Gemfile.local

```diff
@@ -1 +1,3 @@
 gem 'pg'
+gem 'tzinfo-data'
```

Comment:
Not sure about upstream. tzinfo couldn't find timezone data, this gem gives it the data.